store/index.js

```diff
@@ -537,6 +537,10 @@ export const actions = {
       promises['settings'] = dispatch('management/findAll', { type: MANAGEMENT.SETTING });
     }
 
+    if ( getters['management/canList'](MANAGEMENT.GLOBAL_ROLE_BINDING) ) {
```

Comment:
Ok, I see the gotcha here, I failed to realise we'd need to fetch all global role bindings up front.

Need to decide which is preferable to resolve https://github.com/rancher/dashboard/issues/4954

1. Load global role bindings on every visit 
- could impact systems with lots of people/project 
- could be a nice feature for other use cases
2. Show Global Settings to everyone
- reveal admin only things to non-admins 
- they can do this already, but not in the UI